src/test/java/com/flightstats/hub/app/HubPropertiesTest.java

```diff
@@ -10,9 +10,10 @@
     @Test
     public void testIsReadOnly() {
         assertFalse(HubProperties.isReadOnly());
-        HubProperties.setProperty("hub.read.only", "somewhere," + HubHost.getLocalName());
+        HubProperties.setProperty("hub.read.only", "true");
         assertTrue(HubProperties.isReadOnly());
         HubProperties.setProperty("hub.read.only", "");
```

Comment:
I know this is an existing test, but I'm a big fan of only having one test case (and preferably one assertion) per test.

Reply:
Yeah, same here.  I'll fix it since I'm looking at it now.  :)